workshop/main.rb

```diff
@@ -0,0 +1,50 @@
+require_relative "./selenium.rb"
+class LoginFlow
+    extend SeleniumFramework
+
+    def self.login
+        setup("https://www.cocoleni.in/")
+        sleep(15)
+        chk = checkDisplay(:class,"mc-modal-bg")
+        if chk == true
+            clickElement(:class,"mc-closeModal")
+        end
+ 		clickElement(:xpath,"/html/body/header/nav/div/div/div/div[2]")
+ 		enterText(:id,"LoginForm_username","test@mail.com")
+ 		enterText(:id,"LoginForm_password","ttestword")
+ 		clickElement(:name,"yt0")
+ 		done
+    end
+
+    def self.signup
+        setup("https://www.cocoleni.in/")
+        chk = checkDisplay(:class,"mc-modal-bg")
+        if chk == true
+            clickElement(:class,"mc-closeModal")
+        end
+ 		clickElement(:xpath,"/html/body/header/nav/div/div/div/div[2]")
+ 		enterText(:id,"User_email","test@mail.com")
+ 		enterText(:id,"password","ttestword")
+ 		clickElement(:name,"yt1")
+ 		done
+    end
+
+    def self.search
+        setup("https://www.cocoleni.in/")
+        chk = checkDisplay(:class,"mc-modal-bg")
+        if chk == true
```

Comment:
Since chk is a boolean variable you do not need to perform an additional equality check